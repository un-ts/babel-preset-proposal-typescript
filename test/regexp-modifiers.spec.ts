import { execute } from './helpers.js'
import _ from './regexp-modifiers.js'

const proposal = 'regexp-modifiers'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(_('Aa')).toEqual(true)
  expect(_('Ba')).toEqual(false)
})
