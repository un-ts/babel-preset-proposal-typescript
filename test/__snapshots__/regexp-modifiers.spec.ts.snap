// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`regexp-modifiers 1`] = `"Invalid regular expression: /(?i:a)a/: Invalid group"`;
